### PR TITLE
Fixed velocityprofile_dirac does not stop bug and replaced tab with 4…

### PR DESCRIPTION
**Pull request by zchen24**

### CODE REVIEW

**Comment by smits** [comment]
Looks good, it would be nice if you could add a unittest for this.

**Comment by zchen24** [comment]
@smits Thanks for reviewing. A unit test has been added.

**Comment by zchen24** [comment]
@smits Could you please take a look at the added unit-test? Thanks.

File: orocos_kdl/tests/velocityprofiletest.cpp [hunk 1]
```diff
@@ -334,3 +334,45 @@ void VelocityProfileTest::TestTrapHalf_SetDuration_End()
...
+    v.SetProfile(pos1, pos2);
+
+    time = 0;
+    CPPUNIT_ASSERT_DOUBLES_EQUAL(pos1, v.Pos(time), epsilon);
+
+    time = 1.0;
+    CPPUNIT_ASSERT_DOUBLES_EQUAL(pos2, v.Pos(time), epsilon);
+
+    time = -1.0;
+    CPPUNIT_ASSERT_DOUBLES_EQUAL(pos2, v.Pos(time), epsilon);
```
Comment (smits) [inline orocos_kdl/tests/velocityprofiletest.cpp]: Shouldn't this be `pos1`?

Comment (zchen24) [reply orocos_kdl/tests/velocityprofiletest.cpp]: @smits Updated the source code and unit-test.

**Pull request merged**